app/helpers/application_helper/toolbar/vm_infras_center.rb

```diff
@@ -1,4 +1,13 @@
 class ApplicationHelper::Toolbar::VmInfrasCenter < ApplicationHelper::Toolbar::Basic
+  button_group('vms_reload', [
+    button(
+      :vms_reload,
+      'fa fa-refresh fa-lg',
+      N_('Refresh selected Reports'),
```

Comment:
This string needs to change as does the one below.

Reply:
Should be fixed now @Fryguy